Arch linux / error while loading shared libraries: libbotan-3.so.10: cannot open shared object file: No such file or directory

### Have you searched for an existing issue?

- [x] Yes, I tried searching and reviewed the pinned issues

### Brief Summary

After updating my cachyos  keepassxc from develop branch stopped working due to incompatible botan dependency.

```
❯keepassxc
keepassxc: error while loading shared libraries: libbotan-3.so.10: cannot open shared object file: No such file or directory
```

When trying to recompile from develop branch:
```
/home/michau/bin/keepassxc/src/sshagent/OpenSSHKeyGen.cpp: In function ‘bool OpenSSHKeyGen::generateECDSA(OpenSSHKey&, int)’:
/home/michau/bin/keepassxc/src/sshagent/OpenSSHKeyGen.cpp:91:43: error: variable ‘Botan::EC_Group domain’ has initializer but incomplete type
   91 |             Botan::EC_Group domain(QString("secp%1r1").arg(bits).toStdString());
      |                                           ^
make[2]: *** [src/sshagent/CMakeFiles/sshagent.dir/build.make:184: src/sshagent/CMakeFiles/sshagent.dir/OpenSSHKeyGen.cpp.o] Błąd 1
make[2]: *** Oczekiwanie na niezakończone zadania....
```

Using botan 3.11.0-1.1.
```
❯ pacman -Qi botan
Zainstalowano z       : cachyos-extra-v3
Nazwa                 : botan
Wersja                : 3.11.0-1.1
Opis                  : Crypto library written in C++
Architektura          : x86_64_v3
Adres url             : https://botan.randombit.net/
Licencje              : BSD-2-Clause
Grupy                 : Brak
Dostarcza             : Brak
Zależy od             : bzip2  gcc-libs  glibc  sqlite  xz  zlib
Opcjonalne zależności : boost-libs: for the botan executable [zainstalowano]
                        python: for using botan2.py [zainstalowano]
Wymagany przez        : Brak
Opcjonalny dla        : qca-qt6
Konfliktuje z         : Brak
Zastępuje             : Brak
Rozmiar po instalacji : 13,20 MiB
Autor pakietu         : CachyOS <admin@cachyos.org>
Data zbudowania       : pon, 16 mar 2026, 00:46:45
Data zainstalowania   : pon, 16 mar 2026, 13:10:27
Powód zainstalowania  : Ręczny
Skrypt instalacyjny   : Nie
Potwierdzone przez    : Podpis
```

I fixed this issue by adding `#include <botan/ec_group.h>` in `OpenSSHKeyGen.cpp`. But I'm not C++ dev.

### Expected Versus Actual Behavior

_No response_

### Steps to Reproduce

1. Using cachy os use sudo pacman -Syu
2. keepassxc from develop branch will stop running.

### KeePassXC-Browser Debug Information

```Text

```

--- a/src/sshagent/OpenSSHKeyGen.cpp
+++ b/src/sshagent/OpenSSHKeyGen.cpp
@@ -25,8 +25,8 @@
 #include <botan/ed25519.h>
 #include <botan/rsa.h>
 #include <botan/version.h>
-#if BOTAN_VERSION_CODE >= BOTAN_VERSION_CODE_FOR(3,11,0)
-  #include <botan/ec_group.h>
+#if BOTAN_VERSION_CODE >= BOTAN_VERSION_CODE_FOR(3, 11, 0)
+#include <botan/ec_group.h>
 #endif
 
 namespace OpenSSHKeyGen

--- a/src/sshagent/OpenSSHKeyGen.cpp
+++ b/src/sshagent/OpenSSHKeyGen.cpp
@@ -24,6 +24,10 @@
 #include <botan/ecdsa.h>
 #include <botan/ed25519.h>
 #include <botan/rsa.h>
+#include <botan/version.h>
+#if BOTAN_VERSION_CODE >= BOTAN_VERSION_CODE_FOR(3,11,0)
+  #include <botan/ec_group.h>
+#endif
 
 namespace OpenSSHKeyGen
 {

